### PR TITLE
Feature/loop 58 configurable logout button

### PR DESCRIPTION
**Pull request by syastrov**

### CODE REVIEW

File: loopdk.make [hunk 1]
```diff
@@ -172,6 +172,11 @@ projects[saml_sp][subdir] = "contrib"
 projects[saml_sp][download][type] = "git"
 projects[saml_sp][download][url] = "http://git.drupal.org/project/saml_sp.git"
 projects[saml_sp][download][branch] = "7.x-2.x"
+; Important, so we can check if a user is signed in via SAML SP:
+projects[saml_sp][patch][] = "https://www.drupal.org/files/issues/2649458-saml_sp-is_user_authenticated_via_saml-2.patch"
+; SAML SP bugfixes:
+projects[saml_sp][patch][] = "https://www.drupal.org/files/issues/nameid-correction.patch"
```
Comment (rimi-itk) [inline loopdk.make:178]: Are these (last two) patches needed to check if a user is logged in via saml, or are they more general fixes?

File: modules/loop_configure_theme/loop_configure_theme.module [hunk 2]
```diff
@@ -5,6 +5,26 @@
  * Add fields to a form, letting the administrator choose the header color and logo height.
  */
 function loop_configure_theme_form_system_theme_settings_alter(&$form, $form_state) {
+  $form['logout_settings'] = array(
```
Comment (rimi-itk) [inline modules/loop_configure_theme/loop_configure_theme.module]: This should be moved to the Loop theme settings (themes/loop/theme-settings.php).

**Comment by syastrov** [comment]
This one is to be able to check if a user is logged in via saml:

```
projects[saml_sp][patch][] = "https://www.drupal.org/files/issues/2649458-saml_sp-is_user_authenticated_via_saml-2.patch"
```

This one is a fix, because I couldn't get the module to work (I received an error about an undefined field [see https://www.drupal.org/node/2572715]):

```
projects[saml_sp][patch][] = "https://www.drupal.org/files/issues/nameid-correction.patch"
```

**Comment by rimi-itk** [comment]
This looks good. Please move the settings to the Loop theme settings.

**Comment by syastrov** [comment]
The settings have been moved now to the theme-settings.php file.

**Pull request merged**